docs/serving/samples/grpc-ping-go/README.md

```diff
@@ -2,7 +2,7 @@ A simple gRPC server written in Go that you can use for testing.
 
 ## Prerequisites
 
-- [Install the Knative Serving version 0.4 or later](../../../install/README.md).
+- [Install the Knative Serving version 0.6 or later](../../../install/README.md).
```

Comment:
we could remove the version entirely that way it won't get out of date and always reference the latest install doc.

Reply:
👌

Reply:
my only concern here is if someone with an existing older version of Serving (pre 0.6) tries to run this

what if we say the "latest version?

```suggestion
- [Install the latest version of Knative Serving](../../../install/README.md).
```